src/examples/CapiResize.cpp

```diff
@@ -0,0 +1,125 @@
+#include <Elementary.h>
+#include <thorvg_capi.h>
+
+#include <stdlib.h>
+
+#define WIDTH 400
+#define HEIGHT 400
+
+
+/************************************************************************/
+/* Capi Test Code                                                       */
+/************************************************************************/
+
+static uint32_t *buffer = NULL;
+
+static Tvg_Paint *shape = NULL;
+static Tvg_Paint *scene = NULL;
+
+static Tvg_Canvas *canvas = NULL;
+
+static Eo *view = NULL;
+
+void init()
+{
+    canvas = tvg_swcanvas_create();
+    scene = tvg_scene_new();
+    shape = tvg_shape_new();
+
+    tvg_paint_set_opacity(shape, 0);
+    tvg_shape_append_rect(shape, 100, 0, 100, 100, 0, 0);
+    tvg_shape_set_stroke_color(shape, 255, 0, 0, 255);
+    tvg_shape_set_stroke_width(shape, 1);
+
+    buffer = (uint32_t*) malloc(sizeof(uint32_t) * WIDTH * HEIGHT);
+    tvg_swcanvas_set_target(canvas, buffer, WIDTH, WIDTH, HEIGHT, TVG_COLORSPACE_ARGB8888);
+
+    tvg_canvas_clear(canvas, false);
+    tvg_scene_clear(scene);
+    tvg_canvas_push(canvas, scene);

```

Comment:
it's recommended to push the scene into canvas after paint is ready to update.
you can push the scene to canvas after you pushed all subsequent paints into the scene.
I found an issue that scene doesn't update any RenderUpdateFlag for this (need to fix it), but this is unlikely to occur any crash issues.

Reply:
Added one issue ticket... https://github.com/Samsung/thorvg/issues/200

Reply:
No... read again... scene doesn't have any issues. forget RenderUpdateFlag stuff here.